collections/infrastructure/roles/dns_client/README.md

```diff
@@ -6,6 +6,14 @@ This role provides a basic `/etc/resolv.conf` file.
 
 ## Instructions
 
+It is possible to force DNS servers by specifying:
+
+```yaml
+dns_client_server:
```

Comment:
Can you juste rename this variable to dns_client_servers (so with an s at end), since this is a list of servers, not a single one.

Reply:
Anyway, you should have also dns_client_external_server to rename with a 's' as it is also a list of servers ;-)